src/Temporalio/Activities/ActivityInfo.cs

```diff
@@ -20,6 +20,14 @@ namespace Temporalio.Activities
     /// <param name="HeartbeatTimeout">Heartbeat timeout set by the caller.</param>
     /// <param name="IsLocal">Whether the activity is a local activity or not.</param>
     /// <param name="Priority">The Priority of this activity.</param>
+    /// <param name="RetryPolicy">
+    /// The retry policy of this activity.
+    /// <para>
+    /// Note that the server may have set a different policy than the one provided when scheduling the activity.
+    /// If the value is null, it means the server didn't send information about retry policy (e.g. due to old server
+    /// version), but it may still be defined server-side.
```

Comment:
```suggestion
    /// version), but it may still be defined server-side. It may also be null if testing with an activity testing environment.
```

Not that important, just something I noticed reading code

Reply:
Or maybe I'll just fill in default retry policy.

Reply:
I think null is ok here. Users can of course customize this if they need to for testing their activity.

Reply:
Changed testing environment to initialize retry policy, see comment at https://github.com/temporalio/sdk-ruby/pull/341#discussion_r2363114625